cirq/experiments/cross_entropy_benchmarking.py

```diff
@@ -25,6 +25,52 @@
 
 CrossEntropyPair = NamedTuple('CrossEntropyPair', [('num_cycle', int),
                                                    ('xeb_fidelity', float)])
+SpecklePurityPair = NamedTuple('SpecklePurityPair', [('num_cycle', int),
+                                                     ('purity', float)])
+
+
+def purity_from_probabilities(
```

Comment:
This function should live in separate file "purity_estimation.py" for consistency with "fidelity_estimation.py".

Reply:
Done.